test/run-tests

```diff
@@ -249,20 +250,32 @@ def run(*argv, env=None, check=True, cwd=None, return_stdout=False):
     return result
 
 
-def origurl(path):
-    """
-    Returns the original URL that a given file was downloaded from.
-    """
-
+def get_metadata_from_file(path, attr_key):
     try:
-        urlbytes = os.getxattr(path, URL_XATTR)
+        urlbytes = os.getxattr(path, attr_key)
```

Comment:
this isn't `urlbytes` anymore - I suggest `mdbytes` for metadata bytes

Reply:
Yes, `mdbytes` is better. I fixed that.